domain/src/main/kotlin/com/yapp/bol/group/member/Members.kt

```diff
@@ -0,0 +1,27 @@
+package com.yapp.bol.group.member
+
+class Members(private val members: MutableList<Member> = mutableListOf()) {
+    private fun addOwner(userId: Long, nickname: String) {
+        val owner = Member.createOwner(userId, nickname)
+
+        members.add(owner)
+    }
+
+    fun toList(): List<Member> {
+        return members.toList()
+    }
+
+    companion object {
+        fun fromList(members: List<Member>): Members {
+            return Members(members.toMutableList())
+        }
+
+        fun of(userId: Long, nickname: String): Members {

```

Comment:
p2. 해당 함수를 보니 Members 전체의 의도가 혼돈이 되네요.

- Members에 2개 이상의 owner 역할이 주어지는 경우 어떻게 처리가 되는건지,
- owner 역할이 없으면 어떻게 되는 건지

p3. 함수 네이밍을 바꾸거나, 파라미터를 바꿀 필요가 있어보입니다.
해당 userId 및 nickname만으로는 Members에 onwer로 등록해준다는 느낌이 나지 않아요.

또한, Member에 의존성이 심하기도 하네요.

```suggestion
        fun of(onwer: Member): Members {
```